lib/dfe/analytics.rb

```diff
@@ -95,6 +95,11 @@ def self.initialize!
         return
       end
 
+      unless File.exist?(Rails.root.join('config/initializers/dfe_analytics.rb'))
+        Rails.logger.info('Warning: DfE Analytics is not set up. Run `bundle exec rails generate dfe:analytics:install')
+        return
+      end
```

Comment:
Just had a second thought on this. Is this required for Production ? And should we write to the terminal ?

```suggestion
      if !Rails.env.production? && !File.exist?(Rails.root.join('config/initializers/dfe_analytics.rb'))
        message = "Warning: DfE Analytics is not set up. Run: 'bundle exec rails generate dfe:analytics:install'"
        Rails.logger.info(message)
        puts message
        return
      end
```

Reply:
Yes, good plan!